### DISCUSSION TITLE
Location of Python API docs?

### DISCUSSION THREAD

**Discussion by mallman** [featureform/featureform#1135] [Q&A]
Are the api docs for the Python SDK available online? I mean the generated python API docs, not https://sdk.featureform.com.

**Comment by ahmadnazeri** [comment]
Hi @mallman, do you mind elaborating please? Currently, we have our [main docs](http://docs.featureform.com) and the [sdk docs](http://sdk.featureform.com).

**Reply by mallman** [reply]
I mean like what you get from

```sh
cd client/src
pdoc featureform
```
